### DISCUSSION TITLE
I am having issues with createModel

### DISCUSSION THREAD

**Discussion by OlaoluwaM** [statelyai/xstate#2407] [Q&A] [answered]
I am using the new `Model` API to mitigate some issues I was having with Typescript when using the `assign` function with an event transition action. However I cannot seem to find the `createMachine` method on my model object. I am also not seeing it in the type definitions for `createModel` in `model.d.ts`. Any thoughts?

**Comment by johtso** [comment]
~~`createMachine` isn't a method of the `Model`.~~
As @christoph-fricke said, it was only added to the model in v4.22.0.

Prior to that you would do:
```ts
import { createMachine } from 'xstate';

createMachine<typeof myModel>(...
```

**Reply by OlaoluwaM** [reply]
Oh okay, then I suggest removing the following from the docs in favor of your example, to avoid future confusion :)

![Screenshot_20210710-200214_Chrome.jpg](https://user-images.githubusercontent.com/37044906/125173931-ee692680-e1b9-11eb-8865-9b75ef840472.jpg)

**Comment by christoph-fricke** [comment, marked as answer]
Do you have the latest version installed? `createMachine` has been added to the model in [v4.22.0](https://github.com/statelyai/xstate/releases/tag/xstate%404.22.0).

**Reply by johtso** [reply]
Oops, my bad! Things change quickly..
